modules/preview-react/select/stories/stories.tsx

```diff
@@ -139,6 +149,13 @@ storiesOf('Preview/Select/React/Top Label', module)
     <FormField label="Label" inputId="select-grow" grow={true}>
       {controlComponent(<Select name="contact" options={options} />)}
     </FormField>
+  ))
+  .add('Default with Custom Render Selected Method', () => (
+    <FormField label="Label" inputId="select-default-custom-selected">
+      {controlComponent(
+        <Select name="icon" options={customOptions} renderSelected={customRenderSelectedOption} />
+      )}
+    </FormField>
```

Comment:
It seems likely that folks would use `renderSelected` and `renderOption` together. Instead of creating this new "Default with Custom Render Selected Method" story, what do you think about showing the `renderSelected` prop using the existing ["Default with Custom Options Data"](https://github.com/giulialubet/canvas-kit/blob/custom-render-selected/modules/preview-react/select/stories/stories.tsx#L109) story? The [`<Select>`](https://github.com/giulialubet/canvas-kit/blob/custom-render-selected/modules/preview-react/select/stories/stories.tsx#L112) in "Default with Custom Options Data" would receive the `renderSelected` prop in addition to the existing `renderOption` prop. The example would then show an icon for the selected option in addition to the icons already shown for the menu options.

The `customRenderSelectedOption` function would remain unchanged (except perhaps renaming it to `customRenderSelected` to be consistent with the rest of my feedback above).